helm_deploy/django-app/values.yaml

```diff
@@ -0,0 +1,33 @@
+# Default values for django-app.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/cloud-platform-demo-app
+  tag: latest
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - helm-demo.apps.cloud-platforms-test.k8s.integration.dsd.io
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+postgresql:
+    postgresUser: "django"
```

Comment:
I'm not 100% sure about this myself, but I think it's probably better to have empty strings as the default here, to force users to set values themselves and not use default credentials.

Reply:
The password for postgres is set at creation time. All you'd be doing is defining the username. For the quickstart are you happy to have the username hardcoded? (maybe make it explicit in user docs)

Reply:
Yep - as discussed in person, given that the intended use case for an in-cluster database is for non-production environments like staging, test etc it's reasonable to have default values in place here.